ambry-store/src/main/java/com.github.ambry.store/HardDeleter.java

```diff
@@ -299,6 +349,14 @@ long getProgress() {
   }
 
   /**
+   * Returns true if the hard delete is currently paused.
+   * @return true if paused, false otherwise.
+   */
+  boolean isPaused() {
```

Comment:
yes. Would prefer to have it as private variable and the tests had to use this method